backend/Tasque.WebAPI/Program.cs

```diff
@@ -40,6 +40,7 @@
 app.UseStaticFiles();
 
 app.UseRouting();
+app.MapControllers();
```

Comment:
MapControllers is an 'Endpoint' type middleware. It must be the last one in the middleware list. Use this article for more details:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-6.0#middleware-order

Reply:
before app.Run()

Reply:
yep